AEPMedia/Sources/MediaCollection/MediaCollectionReportHelper.swift

```diff
@@ -200,8 +200,8 @@ class MediaCollectionReportHelper {
     private static func serializeCustomerId(customerIds: [[String: Any]]) -> [String: [String: Any]] {
         var serializedCustomerIds: [String: [String: Any]] = [:]
         for customerId in customerIds {
-            if let idType = customerId[MediaConstants.MediaCollection.Session.VISITOR_ID_TYPE] as? String, let idValue = customerId[MediaConstants.MediaCollection.Session.VISITOR_ID] as? String, let authState = customerId[MediaConstants.MediaCollection.Session.VISITOR_ID_AUTHENTICATION_STATE] as? Int {
-                serializedCustomerIds[idType] = ["id": idValue, "authState": authState]
+            if let idTyp = customerId[MediaConstants.MediaCollection.Session.VISITOR_ID_TYPE] as? String, let idVal = customerId[MediaConstants.MediaCollection.Session.VISITOR_ID] as? String, let authState = customerId[MediaConstants.MediaCollection.Session.VISITOR_ID_AUTHENTICATION_STATE] as? Int {
```

Comment:
we can use `idType` and `idValue`, but split on new line before each let condition